articles/azure-functions/durable/durable-functions-instance-management.md

```diff
@@ -40,7 +40,9 @@ The parameters to [StartNewAsync](https://azure.github.io/azure-functions-durabl
 * **Input**: Any JSON-serializable data that should be passed as the input to the orchestrator function.
 * **InstanceId**: (Optional) The unique ID of the instance. If you don't specify this parameter, the method uses a random ID.
 
-Here is a simple C# example:
+Here are some examples:
+
+### C#
```

Comment:
Should we specify that this is precompiled C#, or do you think the distinction is only necessary when a precompiled C# sample is presented alongside a C# script sample?

(Applies to any other instances of `### C#` in this PR.)

Reply:
Yeah that was how I made the decision; if a `Script` example was present, I used `Precompiled` otherwise just left it as `C#`